app/gateway/network/dns-config-reference.md

```diff
@@ -147,4 +145,80 @@ configuration accordingly.
 with `Set-Cookie`, they may have since been deleted, or may have expired.
 Review the `Set-Cookie` information to see when the cookie was set to expire
 and subsequent requests to determine if any other response has issued a
-`Set-Cookie` that deleted it (by setting expiration to a date in the past).
\ No newline at end of file
+`Set-Cookie` that deleted it (by setting expiration to a date in the past).
+
+## Migrate to the new DNS client {% new_in 3.8 %}
+
+The 3.8 DNS client introduces a new standardized way to configure a Gateway Service, and helps improve performance.
+When migrating to the new client, review the following changes and make adjustments as necessary.
```

Comment:
```suggestion
When migrating to the new client, review the following changes and make adjustments as necessary.

{:.info}
> While migration isn't necessary and the old DNS client is still supported, we recommend migrating to the new DNS client for better performance and observable statistics via the `/status/dns` API endpoint.
```
Not adding a link for now because there's some contention on where this endpoint needs to live. Angel is working on it.